pkg/system/phase4_configuring.go

```diff
@@ -31,10 +31,16 @@ func (r *Reconciler) ReconcilePhaseConfiguring() error {
 		"noobaa operator started phase 4/4 - \"Configuring\"",
 	)
 
-	if err := r.ReconcileSecretOp(); err != nil {
-		return err
+	if r.JoinSecret == nil {
```

Comment:
Same as the comment on phase2? Maybe also 3 (even though its not a lot of changes)

Reply:
Please see my answer above